src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$size-stopwatch: 80vmin;
+$size-border: 1vmin;
+$minute-rotate: 60s;
+
+.page__body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.stopwatch {
+  $minute-rotate: 60s;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $size-stopwatch;
+  width: $size-stopwatch;
+  border: $size-border dotted #000;
+  border-radius: 50%;
+  transform-origin: bottom;
+
+  &--speed-up {
+    $minute-rotate: 10s;
+  }
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    transform-origin: bottom;
+    animation: hand-rotate calc(#{$minute-rotate} * 60) steps(60) infinite;
+    bottom: 50%;
+    transform: rotate(0);
+  }
+
+  &__seconds-hand {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    position: absolute;
+    transform-origin: bottom;
+    animation: hand-rotate calc(#{$minute-rotate}) linear infinite;
```

Comment:
The animation for the `stopwatch__seconds-hand` should take `60s` for a full circle, but the current calculation uses `calc(#{$minute-rotate})`, which results in `60s`. This is correct for the basic functionality but ensure that the `speed-up` modifier changes this to `10s` as required.